usecases/cluster/delegate.go

```diff
@@ -234,17 +238,20 @@ func (d *delegate) delete(node string) {
 // sortCandidates by the amount of free space in descending order
 //
 // Two nodes are considered equivalent if the difference between their
-// free spaces is less than 4KB.
+// free spaces is less than 32MB.
 // The free space is just an rough estimate of the actual amount.
-// The Lower bound 4KB helps to mitigate the risk of selecting same set of nodes
+// The Lower bound 32MB helps to mitigate the risk of selecting same set of nodes
 // when selections happens concurrently on different initiator nodes.
 func (d *delegate) sortCandidates(names []string) []string {
+	rand.Shuffle(len(names), func(i, j int) { names[i], names[j] = names[j], names[i] })
```

Comment:
why we shuffle here , given we are going to sort ?

Reply:
We need to shuffle because the sorting algorithm may choose a different order, as nodes with a difference in free space of less than 32MB are considered equivalent